shard.yml

```diff
@@ -1,10 +1,10 @@
 name: multi_auth
-version: 0.1.0
+version: 1.0.0
 
 authors:
   - Sergey Makridenkov <s@msa7.ru>
 
-crystal: 0.23.0
+crystal: 0.25.0
```

Comment:
Why not `0.25.1` ?

Reply:
It's purely informational according to https://github.com/crystal-lang/shards/blob/master/SPEC.md#crystal - and breaking change was introduced in 0.25.0 so I guess that makes most sense here